srcalc/src/main/java/gov/va/med/srcalc/vista/RpcVistaPatientDao.java

```diff
@@ -37,8 +37,11 @@ public Patient getPatient(final int dfn)
                 fDuz, RemoteProcedure.GET_PATIENT, Integer.toString(dfn));
         
         // Fields are separated by '^'
-        final String patientName = results.get(0).split("\\^")[0];
-        final Patient patient = new Patient(dfn, patientName);
+        final String[] fieldArray = results.get(0).split("\\^");
```

Comment:
We should wrap all of this parsing in a try/catch and throw an appropriate `DataAccessException` per the method contract. (I should have done this in the first place.)